templates/dt_template/README.md

```diff
@@ -0,0 +1,58 @@
+# RHOS Deployed Topology %NEW_DT_NAME%
+
+**Based on OpenStack K8S operators from the "main" branch of the [OpenStack Operator repo](https://github.com/openstack-k8s-operators/openstack-operator/tree/78b3c876eaf9168f9d95b201997ebdc2da42fa02) on Oct 17th, 2023**
+
+## General information
+
+| Revision | Change                | Date             |
+|--------: | :-------------------- | :--------------: |
+| v0.1     | Initial publication   | %TODAY%      |
+
+## Node topology
+| Node role                                     | bm/vm | amount |
+| --------------------------------------------- | ----- | ------ |
+| Openshift master/worker combo-node cluster    | vm    | 3      |
+| Compute nodes                                 | vm    | 3      |
+| Galera DB node, replicating                   | vm    | 3      |
+| Hardware security module                      | bm    | 1      |
+
+## Services, enabled features and configurations
+| Service                                     | configuration                   | Lock-in coverage?  |
+| ------------------------------------------- | ------------------------------- | ------------------ |
+| RabbitMQ                                    | default                         | Must have          |
+| OVN                                         | default                         | Must have          |
+| RabbitMQ                                    | default                         | Must have          |
+| Cinder                                      | Ceph/RGW backend                | optional           |
+| Manila                                      | Ceph/Ganesha+MDS Backend        | Must have          |
+| Glance                                      | NetApp, Fibre channel           | Must have          |
+| Keystone                                    | Special appliance [redacted]    | Interchangable     |
+| FIPS Mode                                   | default                         | Must have/standard |
+
+## Considerations
+
+1. Glance requirement dictates specific lab
+2. HSM access limited geographically
+
+## Testing tree
+
+| Test framework   | Stage to run | Special configuration | Test case to report |
```

Comment:
"Test case to report" is not needed: it may be a list too big, it may be just changed into a reference to the test run, but as it is not a public information it should not live here

Reply:
How do we capture the info regarding which coverage needs for which team this addresses?

Reply:
Here  - or better, i nthe code which implements the PR - you could see the list of tests executed (if tempest, the path of the tempest tests; if tobiko, same; if horizon scenario tests, again the list of tests or all).